app/containers/GameProfilePage.js

```diff
@@ -6,9 +6,11 @@ import { playFile, setStatsGamePage } from "../actions/fileLoader";
 import { dismissError } from "../actions/error";
 
 function mapStateToProps(state) {
+  const index = state.fileLoader.statsGameIndex;
   return {
     store: state.game,
-    statsGameIndex: state.fileLoader.statsGameIndex,
+    statsGameIndex: index,
+    file: state.fileLoader.files[index],
```

Comment:
a bit concerned about this. wondering if it would crash if opening a file directly from the file system, for example

Reply:
I haven't tested it, but shouldn't that file show up in the ``files`` array?

https://github.com/project-slippi/slippi-desktop-app/blob/b3d0044999f75319c68a12a48ef8ea03a29140ba/app/actions/fileLoader.js#L73-L82

I'm not sure if there's any way to load a file outside of the ``ROOT_FOLDER`` but this should cover the bases if there isn't, no?

Reply:
There is a way, you can double click an slp file from anywhere. One way to test it is to just `yarn package` your branch and install it. I don't remember how to test it in dev mode though I'm sure I've done it before.

Reply:
Just tested it (using files both from root dir and elsewhere). The only thing that happens is that time is set to Unknown again. No crashes, which makes me wonder what's going on. I'll take a deeper look when I figure out how to open console/dev mode when opening a specific file. What I could do is just pass the ``fileLoader`` object and see how to grab the file/attribute that I'm looking for from within the function.

Reply:
No idea how to run with a single file in dev mode (tried setting ``DEBUG_PROD`` to true but couldn't open the console, windows cmd piping also doesn't work) or to see the console, which unfortunately means I don't have enough info to fix this bug. However, it isn't crash-inducing.

Reply:
You might be able to hardcode `argURI` at the following line to point to a replay on your file system temporarily

https://github.com/project-slippi/slippi-desktop-app/blob/master/app/main.dev.js#L374